cmd/git-chglog/main.go

```diff
@@ -83,6 +83,18 @@ func CreateApp(actionFunc cli.ActionFunc) *cli.App {
 			Value: ".chglog/config.yml",
 		},
 
+		// template
+		cli.StringFlag{
+			Name:  "template, t",
+			Usage: "specifies a template file to pick up. If not specified, use the one in config",
+		},
+
+		// repository url
+		cli.StringFlag{
```

Comment:
This will be needed after the rebase with current `master` and the bump to `urfave/cli/v2`
```suggestion
		&cli.StringFlag{
```